docs/ticket-triage.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Ticket Triage
+=============
+
+Ticket triage is a weekly, informal ritual that is meant to solve the problem of
+tickets getting opened and then forgotten about. It is simple and keeps project
+momentum going and prevents ticket cruft.
+
+It fosters conversation around project tasks and philosophies as they relate to 
+milestones.
+
+Process
+-------
+- The role of Ticket Triager rotates weekly 
+- The Triager needs admin status on ``Trac``
+- The Triager looks at all the tickets that have been created in the last week 
+    - They can use a custom query or do this as the week progresses 
+    - BONUS ROUND: Dig up a stale ticket from the past
+- Assign each ticket to a milestone on the Roadmap 
+- The following situations merit discussion:
+    - A ticket doesn't have an appropriate milestone and we should create one 
+    - A ticket, in vanishingly rare circumstances, should be deleted
```

Comment:
Let's see if we can enumerate these circumstances.

* The ticket is spam
* The ticket contains sensitive information and harm will come to one or more people if it continues to be distributed

Anything else come to mind for anyone?

Ticket deletion requires "ticket admin" capabilities in trac.  We might also want to contemplate the fact that we may switch from trac to a system that doesn't support ticket deletion at all.

Reply:
Maybe it's fine as-is: if something seems like it should be deleted, the Triager probably should bring it up for discussion (rather than just deleting it). If it's especially problematic / time-sensitive they could just delete it (but should probably still bring it up).